src/cn-flex-form.service.js

```diff
@@ -734,7 +734,16 @@ function CNFlexFormService(
                   let p = schema && schema.format === 'currency-dollars' ? 2 : 0;
 
                   if(adjustment.math[1] === '*') {
-                    result = _.floor(result, p);
+                    let bottom = _.floor(result, p);
+                    if(bottom % 10 == 9) {
+                      result = bottom + 1;
+                    }
+                    else if(bottom % 10 == 1) {
+                      result = bottom - 1;
+                    }
+                    else {
+                      result = bottom;
+                    }
                   }
                   else if(adjustment.math[1] === '/') {
                     result = _.ceil(result, p);
```

Comment:
Shouldn't we do the same for division? We multiply to get total budget from media budget and divide for vice versa. A change in one will trigger a change in the other so if the roundtrip calculations aren't the same this can cause a loop where the budgets veer way off from the input value.

I.e., if a media budget is set, that'll trigger the total budget to be calculated and set, that will then trigger the media budget to be calculated and set, and if it's different from its current value because we're not applying the same fudging, then we can into an update loop as well as end up with values that are incorrect.

Reply:
There will always be some values for which the roundtrip calculation will not end where it began, no matter what kind of rounding logic we use. All we can do is try to choose which values that will be true for. This implementation ensures that all multiples of 10 cents will end where they started.

Reply:
Also it seems that the behavior is only to update in one direction then stop.